lms/djangoapps/ccx/views.py

```diff
@@ -489,30 +511,28 @@ def ccx_student_management(request, course, ccx=None):
 
     action = request.POST.get('student-action', None)
     student_id = request.POST.get('student-id', '')
-    user = email = None
-    error_message = ""
     course_key = CCXLocator.from_course_locator(course.id, ccx.id)
-    try:
-        user = get_student_from_identifier(student_id)
-    except User.DoesNotExist:
-        email = student_id
-        error_message = validate_student_email(email)
-        if email and not error_message:
-            error_message = _(
-                'Could not find a user with name or email "{email}" '
-            ).format(email=email)
-    else:
-        email = user.email
-        error_message = validate_student_email(email)
-
-    if error_message is None:
-        if action == 'add':
-            # by decree, no emails sent to students added this way
-            # by decree, any students added this way are auto_enrolled
-            enroll_email(course_key, email, auto_enroll=True, email_students=False)
-        elif action == 'revoke':
-            unenroll_email(course_key, email, email_students=False)
-    else:
+    error_message = None
+    email_params = get_email_params(course, auto_enroll=True, course_key=course_key, display_name=ccx.display_name)
+
+    if action == 'add':
+        # by decree, no emails sent to students added this way
+        # by decree, any students added this way are auto_enrolled
+        try:
+            _enroll_students(
+                identifiers=[student_id], course_key=course_key, email_students=False, email_params=email_params)
+        except (CCXCourseFullException, CCXUserValidationException) as exp:
+            error_message = "{0}".format(exp)
```

Comment:
can you instead make it `log.info("%s", exp)`? See http://stackoverflow.com/a/4149231/2669626